python/lsst/summit/extras/fastStarTrackerAnalysis.py

```diff
@@ -43,15 +43,18 @@
 from lsst.utils.iteration import ensure_iterable
 
 __all__ = (
+    "getStreamingSequences",
     "getFlux",
     "getBackgroundLevel",
     "countOverThresholdPixels",
     "sortSourcesByFlux",
     "findFastStarTrackerImageSources",
     "checkResultConsistency",
     "plotSourceMovement",
-    "plotSource",
     "plotSourcesOnImage",
+    "plotSource",
+    "Source",
+    "NanSource",
```

Comment:
If you're sorting by order of appearance in the file, these last two need to move up above `findFastStarTrackerImageSources`.